src/coredump/coredumpctl.c

```diff
@@ -913,6 +971,10 @@ int main(int argc, char *argv[]) {
                 log_debug("Journal filter: %s", filter);
         }
 
+        units_active = check_units_active();
+        if (units_active < 0)
+                log_warning("Failed to check if any systemd-coredump@.service units are running.");
```

Comment:
Indeed. check_units_active already logs on all errors, so I removed this and just tweaked the message in check_units_active a bit.